TinkGraphicsContract1.py

```diff
@@ -0,0 +1,235 @@
+            Green = screen.get_at((X, Y)).g
+            Blue = screen.get_at((X, Y)).b
+            # If the rgb values are all above 150 divide them by 3. This makes the light areas darker.
+            if Red > 150 and Green > 150 and Blue > 150:
+                pxarray[X, Y] = (Red/3, Green/3, Blue/3)
+
+# Makes the bridge slightly darker
+def BridgeLightsEffect(Bridge):
+    for Y in xrange(HEIGHT):
+        for X in xrange(WIDTH):
+            Red = screen.get_at((X, Y)).r
+            Green = screen.get_at((X, Y)).g
+            Blue = screen.get_at((X, Y)).b
+            if 100 > Red > 10 and 100 > Green > 10 and 100 > Blue > 10:
+                # - 10 from each rgb value to make it slightly darker
+                pxarray[X, Y] = (Red - 10, Green - 10, Blue - 10)
+
+# blits the images to the screen
+def drawBridge():
+    screen.blit(Bridge, [0, 0])
+
+def drawWreckage():
+    screen.blit(Wreckage, [0, 0])
+
+def drawBiodome():
+    screen.blit(Biodome, [0, 0])
+    pygame.display.flip()
+    Biodome.set_alpha(alpha)
+
+# status of bridge lights
+status = 1
+on = 1
+off = 0
+
+# Setting different transparency values
+alpha = 255
+alpha0 = 255
+alpha1 = 210
+alpha2 = 170
+alpha3 = 130
+alpha4 = 90
+alpha5 = 50
+
+# Setting different background colours
+BackColour = (255, 255, 255)
+WHITE = (255, 255, 255)
+RED = (255, 0, 0)
+BLUE = (0, 188, 255)
+BLACK = (0, 0, 0)
+colour1 = (5, 255, 43)
+colour2 = (242, 252, 45)
+colour3 = (247, 27, 247)
+
+# Wreckage image functions
+
+updates = 0
+
+def darkenAll(n):
+    updates = n
+    for y in xrange(1, HEIGHT - 1):
+        for x in xrange(1, WIDTH - 1):
+            if updates <= 10:
+                    darkenPixel(x, y, updates)
+            else:
+                newUpdates = (20 - updates)
+                redUpdates = (updates - 10)
+                darkenPixel(x, y, newUpdates)
+                reddenPixel(x, y, redUpdates)
+
+def darkenPixel(x, y, n):
+    darkenAmount = 10 * n
+    red = screen.get_at((x, y)).r
+    green = screen.get_at((x, y)).g
+    blue = screen.get_at((x, y)).b
+    if red > darkenAmount:
+        red -= darkenAmount
+    if green > darkenAmount:
+        green -= darkenAmount
+    if blue > darkenAmount:
+        blue -= darkenAmount
+    screen.set_at((x, y), (red, green, blue))
+
+def reddenPixel(x, y, n):
+    Amount = 10 * n
+    Limit = 255 - Amount
+    red = screen.get_at((x, y)).r
+    green = screen.get_at((x, y)).g
+    blue = screen.get_at((x, y)).b
+    if red <= Limit:
+        red += Amount
+    screen.set_at((x, y), (red, green, blue))
+
+def alienTorch():
+    (mousePosX, mousePosY) = pygame.mouse.get_pos()
+    torchRadius = 40
+    for y in xrange(1, HEIGHT - 1):
+        for x in xrange(1, WIDTH - 1):
+            if ((x - mousePosX)**2 + (y - mousePosY)**2) <= (torchRadius**2):
+                screen.set_at((x, y),(Wreckage.get_at((x, y))))
+
+while True:
+    key_pressed = pygame.key.get_pressed()
+
+    if key_pressed[pygame.K_1]:

```

Comment:
Consider the use of a dictionary and function names to eliminate some/all of these if-statements.